app/src/main/java/org/schabi/newpipe/player/helper/PlayerDataSource.java

```diff
@@ -31,11 +31,13 @@
     private static final int MANIFEST_MINIMUM_RETRY = 5;
     private static final int EXTRACTOR_MINIMUM_RETRY = Integer.MAX_VALUE;
 
+    private final Integer continueLoadingCheckIntervalBytes;
```

Comment:
Why is this a non primitive datatype?
Why not use ``int`` instead?

Reply:
Good catch, fixed.